vita3k/motion/src/motion.cpp

```diff
@@ -99,8 +99,12 @@ void refresh_motion(MotionState &state, CtrlState &ctrl_state) {
         }
     }
 
-    if (!found_accel && !found_gyro)
-        return;
+    if (!found_accel && !found_gyro) {
+        if (ctrl_state.udp_client.IsConnected())
+            ctrl_state.udp_client.GetGyroAccel(gyro, gyro_timestamp, accel, accel_timestamp);
+        else
+            return;
+    }
```

Comment:
```suggestion
    if (!found_accel && !found_gyro) {
-        if (ctrl_state.udp_client.IsConnected())
-            ctrl_state.udp_client.GetGyroAccel(gyro, gyro_timestamp, accel, accel_timestamp);
-        else
+        if (!ctrl_state.udp_client.IsConnected()) {
             return;
+        }
+        ctrl_state.udp_client.GetGyroAccel(gyro, gyro_timestamp, accel, accel_timestamp);
+        // udp client already has the correct magnitude.
+        accel *= -SDL_STANDARD_GRAVITY;
     }
```

Cemu hook is based on dsu. It already provides gyro and accelerometer in the correct magnitude. The motion code divides the accelerometer by gravity. So we need to do the inverse here to keep the values the same.